src/OpenTelemetry/Metrics/Exemplar/ExemplarReservoir.cs

```diff
@@ -18,7 +18,7 @@ namespace OpenTelemetry.Metrics;
 /// <summary>
 /// The base class for defining Exemplar Reservoir.
 /// </summary>
-public abstract class ExemplarReservoir
+internal abstract class ExemplarReservoir
```

Comment:
Is this just for now (and later allow custom reservoirs) or the goal is to only allow built-in reservoirs?

Reply:
just for now! 
Reservoirs would be opened up like Filters.